### DISCUSSION TITLE
Issue with hover statement after using bg-white

### DISCUSSION THREAD

**Discussion by wthks** [tailwindlabs/tailwindcss#5780] [Help] [answered]
I tried today the latest alpha tailwindcss version everything works fine without issues but then I found something weird.
when I change the background color of a button to white `bg-white` then the hover color is not working anymore.

```html
<button class="bg-white hover:bg-black">Test</button> // not working
<button class="bg-black hover:bg-white">Test</button> // working
```

I figured this out after deploying my site on my server. on **play.tailwind.css** everything works.

**Temporary Fix**
I added in the tailwind config a new color. so maybe there is something wrong with `bg-white`
```css
tailwind.config = {
        theme: {
          extend: {
            colors: {
	        twWhite: '#FFFFFF'
            },
          },
        },
      }
```

**Comment by NovusOpus** [comment, marked as answer]
Working fine with [Tailwind Play](https://play.tailwindcss.com/nlVgF94jAR) with Version 3.0.0-Alpha.1.

Maybe it's a problem during the building process? or env related issue?

**Edit:**

Also tried with the new JIT CDN, working fine as well:
```HTML
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
    <script src="https://cdn-tailwindcss.vercel.app/"></script>
  </head>
  <body>
    <div class="bg-blue-800 w-full h-screen flex items-center justify-center space-x-4">
        <button class="bg-white hover:bg-black hover:text-white p-6">Test</button>
        <button class="bg-black hover:bg-white text-white hover:text-black p-6">Test</button>
    </div>
  </body>
</html>
```
Also working fine locally, tested with **NextJS** And also with Vite+Vue.

**Reply by wthks** [reply]
Okay. I figured out that the <script> tag should be over everything now it works too. thx for the code snippet.
